spec/controllers/lists/entities_controller_spec.rb

```diff
@@ -0,0 +1,100 @@
+describe Lists::EntitiesController, type: :controller do
```

Comment:
Great tests! Once we have more confidence that js tests run reliably-- we could consider testing most of this with the js/feature tests